pyspark/bigdl/nn/keras/topology.py

```diff
@@ -200,6 +204,9 @@ class Sequential(KerasModel):
 
     >>> sequential = Sequential(name="seq1")
     creating: createKerasSequential
+
+    .. note:: `bigdl.nn.keras` is deprecated in 0.11.
+    This will be removed in future releases.
     """
     def __init__(self, jvalue=None, **kwargs):
```

Comment:
Do we need to add a warning in init?

Reply:
In general no - Analytics Zoo will also used these classes, yes?

Reply:
No. For the Python side, Analytics Zoo's all keras layers are newly wrapped and have no relationship with bigdl's keras layers. The dependencies only exist on Scala side.

Reply:
Done. On the scala side, only deprecate Sequential, Model, and Input class. On the python side, add a warning for the Sequential and Model class.